packages/compass-query-history/src/components/query/query.jsx

```diff
@@ -34,32 +55,40 @@ class Query extends PureComponent {
     const { attributes } = this.props;
 
     return (
-      <li key={index}>
-        <label
+      <div
+        data-test-id="query-history-query-attribute"
+        className={queryAttributeStyles}
+        key={index}
+      >
+        <Label
           data-test-id="query-history-query-label"
-          className={styles.label}>{attrKey}</label>
+          className={labelStyles}
+          htmlFor={`query-history-query-attr-${attrKey}`}
+        >{attrKey}</Label>
```

Comment:
This Code component you are referring in the for here is not an input, right? [HTML label elements have a very particular use-case and meaning](https://html.spec.whatwg.org/multipage/forms.html#the-label-element) and should provide labels for a specific set of elements

Reply:
Thanks for catching! Updated